FlickrApp/app/src/main/java/com/example/flickrapp/FlickrDataRepository.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.flickrapp
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import retrofit2.Call
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class FlickrDataRepository {
+
+    val retrofit: Retrofit =
+        Retrofit.Builder().baseUrl("https://www.flickr.com/services/feeds/")
+            .addConverterFactory(GsonConverterFactory.create()).build()
+
+    val flickrDataWebservice: FlickrDataWebservice =
+        retrofit.create(FlickrDataWebservice::class.java)
+
+    fun getFlickrData(): LiveData<ArrayList<FlickerData>> {
+        val data = MutableLiveData<ArrayList<FlickerData>>()
+        flickrDataWebservice.getFlickrData("json", "en-us", "android,oreo", 1).enqueue(
```

Comment:
rule of thumb is never have magic data in your app, define these as constants and pass them in. It is clearer what they do and easier to maintain when you do this.